.github/workflows/release.yml

```diff
@@ -0,0 +1,135 @@
+# SPDX-FileCopyrightText: 2023 Forschungszentrum Jülich GmbH (FZJ)
+#
+# SPDX-License-Identifier: CC0-1.0
+
+# SPDX-FileContributor: Oliver Bertuch
+
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+    tags:
+      - 'v**'
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        build toml-cli
+        --user
+    - name: Reset package name and version for dev snaps
+      if: github.ref_name == 'develop'
+      run: |
+        toml set --toml-path pyproject.toml tool.poetry.name hermes-test
+        VERSION=$(toml get --toml-path pyproject.toml tool.poetry.version)
+        toml set --toml-path pyproject.toml tool.poetry.version "$VERSION.dev${{ github.run_number }}"
```

Comment:
We should probably decide if we are fine with this numerical for dev snaps.
We can make them PR numbers, too. These numbers can be extracted from Git commit messages.

Reply:
Consensus: we will remove TestPyPI releases and let people know to install dev snaps using `git+http`.